src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -211,6 +213,28 @@ public static PropertyName parsePropertyName(String propertyName) throws ParseEx
         return new PropertyName(trimmedPropertyName);
     }
 
+    /**
+     * Parses a {@code String ownerName} and a {@code String ownerPhone} into an {@code Owner}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code ownerName} or {@code ownerPhone} is invalid.
+     */
+    public static Owner parseOwner(String ownerName, String ownerPhone) throws ParseException {
```

Comment:
Is this method used? I used the existing methods `parseName`and `parsePhone` to parse `ownerName` and `ownerPhone`.

Reply:
Ah yes, I forgot I could use those methods! Oh well...